topcoffea/cfg/data_samples_NDSkim.cfg

```diff
@@ -41,39 +41,48 @@ root://deepthought.crc.nd.edu/
-
-../../topcoffea/json/data_samples/2017/MuonEG_B-UL2017_NDSkim.json
-../../topcoffea/json/data_samples/2017/MuonEG_C-UL2017_NDSkim.json
-../../topcoffea/json/data_samples/2017/MuonEG_D-UL2017_NDSkim.json
-../../topcoffea/json/data_samples/2017/MuonEG_E-UL2017_NDSkim.json
-../../topcoffea/json/data_samples/2017/MuonEG_F-UL2017_NDSkim.json
-
-../../topcoffea/json/data_samples/2017/SingleElectron_B-UL2017_NDSkim.json
-../../topcoffea/json/data_samples/2017/SingleElectron_C-UL2017_NDSkim.json
-../../topcoffea/json/data_samples/2017/SingleElectron_D-UL2017_NDSkim.json
-../../topcoffea/json/data_samples/2017/SingleElectron_E-UL2017_NDSkim.json
-../../topcoffea/json/data_samples/2017/SingleElectron_F-UL2017_NDSkim.json
-
-../../topcoffea/json/data_samples/2017/SingleMuon_B-UL2017_NDSkim.json
-../../topcoffea/json/data_samples/2017/SingleMuon_C-UL2017_NDSkim.json
-../../topcoffea/json/data_samples/2017/SingleMuon_D-UL2017_NDSkim.json
-../../topcoffea/json/data_samples/2017/SingleMuon_E-UL2017_NDSkim.json
-../../topcoffea/json/data_samples/2017/SingleMuon_F-UL2017_NDSkim.json
+#root://deepthought.crc.nd.edu/
+#../../topcoffea/json/data_samples/2017/DoubleEG_B-UL2017_NDSkim.json
+#../../topcoffea/json/data_samples/2017/DoubleEG_C-UL2017_NDSkim.json
+#../../topcoffea/json/data_samples/2017/DoubleEG_D-UL2017_NDSkim.json
+#../../topcoffea/json/data_samples/2017/DoubleEG_E-UL2017_NDSkim.json
+#../../topcoffea/json/data_samples/2017/DoubleEG_F-UL2017_NDSkim.json
+#
+#../../topcoffea/json/data_samples/2017/DoubleMuon_B-UL2017_NDSkim.json
+#../../topcoffea/json/data_samples/2017/DoubleMuon_C-UL2017_NDSkim.json
+#../../topcoffea/json/data_samples/2017/DoubleMuon_D-UL2017_NDSkim.json
+#../../topcoffea/json/data_samples/2017/DoubleMuon_E-UL2017_NDSkim.json
+#../../topcoffea/json/data_samples/2017/DoubleMuon_F-UL2017_NDSkim.json
+#
+#../../topcoffea/json/data_samples/2017/MuonEG_B-UL2017_NDSkim.json
+#../../topcoffea/json/data_samples/2017/MuonEG_C-UL2017_NDSkim.json
+#../../topcoffea/json/data_samples/2017/MuonEG_D-UL2017_NDSkim.json
+#../../topcoffea/json/data_samples/2017/MuonEG_E-UL2017_NDSkim.json
+#../../topcoffea/json/data_samples/2017/MuonEG_F-UL2017_NDSkim.json
+#
+#../../topcoffea/json/data_samples/2017/SingleElectron_B-UL2017_NDSkim.json
+#../../topcoffea/json/data_samples/2017/SingleElectron_C-UL2017_NDSkim.json
+#../../topcoffea/json/data_samples/2017/SingleElectron_D-UL2017_NDSkim.json
+#../../topcoffea/json/data_samples/2017/SingleElectron_E-UL2017_NDSkim.json
+#../../topcoffea/json/data_samples/2017/SingleElectron_F-UL2017_NDSkim.json
+#
+#../../topcoffea/json/data_samples/2017/SingleMuon_B-UL2017_NDSkim.json
+#../../topcoffea/json/data_samples/2017/SingleMuon_C-UL2017_NDSkim.json
+#../../topcoffea/json/data_samples/2017/SingleMuon_D-UL2017_NDSkim.json
+#../../topcoffea/json/data_samples/2017/SingleMuon_E-UL2017_NDSkim.json
+#../../topcoffea/json/data_samples/2017/SingleMuon_F-UL2017_NDSkim.json
```

Comment:
Same comment as above: why are we commenting these JSONs? I think I'd prefer to leave all of the Run 2 JSONs uncommented, since we'll most often be running over the full Run 2 together.

Reply:
fixed with 80bbed74e99fe51bc931a15c9f7ddaec83a66c67